app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder:'name' %>
+  </div>
+
+  <div class="field">
+   <br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder:'email' %>
+  </div>
+
```

Comment:
- [ ] As mentioned in the previous code review, we were asked to modify the views for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A).
However, i think the login view is good since we have the `email` and `password` fields present. For the registration view, i think we are missing the fields for the `photo` and `Bio` although it was not explicitly defined in the previous review but from the ERD diagram